wweval/R/score_real_time_outputs.R

```diff
@@ -0,0 +1,289 @@
+#' Load in and score the real-time outputs
+#'
+#' @param score_type A string indicating which score to generate, either
+#' "crps" or "wis". Note, if using crps, will score draws from nowcast and
+#' forecast. If using wis, will score only the forecasts.
```

Comment:
This is because the hub itself did not accept nowcasts, yes? Might be worth clarifying.

Reply:
Yes